README.md

```diff
@@ -59,4 +59,20 @@ Usage
         job.setInputFormatClass(PravegaInputFormat.class);
 
         // FYI, Key class is 'EventKey', but you won't need it at most of the time.
+
+```
+Or builder style
+```
+        Configuration conf = PravegaInputFormat.configBuilder()
+        // Configuration conf = PravegaInputFormat.configBuilder(conf), if conf is created before
+            .forScope("myScope")
+            .forStream("myStream")
+            .forURI("tcp://127.0.0.1:9090")
```

Comment:
This line made me think about the implementation in the flink-connectors. Would it make sense to have a `PravegaConfig` class like here:

https://github.com/pravega/flink-connectors/blob/master/src/main/java/io/pravega/connectors/flink/PravegaConfig.java#L115

We should also try to unify the naming. Over there, we use `withControllerUri`.

Reply:
accepted